packages/xod-arduino/src/constants.js

```diff
@@ -5,4 +5,8 @@ export const LIVENESS = {
   SIMULATION: 'SIMULATION',
 };
 
+export const SPECIAL_STRING_LITERAL_REPLACEMENTS = {
+  XOD_USERNAME: 'XOD_USERNAME',
+};
```

Comment:
This mapping looks singular. Does it ever have a chance to include not a 1-to-1 mapping?